BlueprintUI/Sources/Element/Element.swift

```diff
@@ -61,6 +61,14 @@ public protocol Element {
     ///                            `subtreeExtent` will be nil if there are no children.
     ///
     /// - Returns: An optional `ViewDescription`.
-    func backingViewDescription(bounds: CGRect, subtreeExtent: CGRect?) -> ViewDescription?
+    func backingViewDescription(bounds: CGRect, subtreeExtent: CGRect?, with context: ElementContext) -> ViewDescription?
```

Comment:
I assume just adding the context here so refactoring is easier? Would be nice to make this one param...

Reply:
(Looks like there's 134 of these in SPOS)

```
~/Desktop/Development/Register1(kve/bump-listable-jun-15-2021):0 git grep "func backingViewDescription(bounds:" | wc -l
     134
```

Reply:
(Looks like about 120 of these return views; the rest return nil.)